frontend/sportsmatch-app/src/App.tsx

```diff
@@ -22,13 +23,15 @@ function App() {
           element={<Wrapper child={<AllSportsList />} />}
```

Comment:
Add something like this here and just take the token from the local storage

`import { OpenAPI } from './generated';

const getToken = async () => {
// return token from local storage.
};

OpenAPI.TOKEN = getToken;`